Gifski/Vendor/CircularProgress.swift

```diff
@@ -4,7 +4,8 @@ import Cocoa
 @IBDesignable
 public final class CircularProgress: NSView {
 	private var lineWidth: Double = 2
-	private lazy var radius = bounds.width < bounds.height ? bounds.midX * 0.8 : bounds.midY * 0.8
+	// TODO: Remove the closure here when targeting Swift 5
+	private lazy var radius: CGFloat = { bounds.width < bounds.height ? bounds.midX * 0.8 : bounds.midY * 0.8 }()
```

Comment:
I got a really weird compile error here so had to make it a closure.

I got this before:

> 1.	While emitting SIL for getter for radius at /Users/sindresorhus/dev/oss/gifski-app/Gifski/Vendor/CircularProgress.swift:8:19
error: Segmentation fault: 11

Reply:
That's odd.